docs/release-notes/release-notes-0.18.5.md

```diff
@@ -30,7 +30,9 @@
 ## RPC Additions
 
 ## lncli Additions
-
+* [`updatechanpolicy`](https://github.com/lightningnetwork/lnd/pull/8805) will
```

Comment:
This was moved from 0.19 and should probably now be removed there. Same with the other one below.